protocols/v2/codec-sv2/src/encoder.rs

```diff
@@ -1,3 +1,47 @@
+//! # Stratum V2 Codec Encoder
+//!
+//! This module provides functionality for encoding Stratum V2 messages, including support for the
+//! Noise protocol for secure communication.
+//!
+//! ## Features
```

Comment:
I usually feel the word "Feature" can be confusing, because in the context of Rust it usually means "feature flag" for `cargo`.